RedM.External.csproj

```diff
@@ -32,7 +32,7 @@
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="CitizenFX.Core">
-      <HintPath>References\CitizenFX.Core.dll</HintPath>
+      <HintPath>..\..\..\..\..\..\RedM\citizen\clr2\lib\mono\4.5\CitizenFX.Core.dll</HintPath>
```

Comment:
Why was this replaced? Make a symbolic link for it. (`mklink/h References\CitizenFX.Core.dll path\to\your\local.dll`)